bench/Alice/Program.cs

```diff
@@ -1,22 +1,26 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
-using System.Text;
 using System.Linq;
 using System.Reflection;
-using System.Diagnostics;
-using System.Collections.Generic;
-
-using NetRegex = System.Text.RegularExpressions.Regex;
-
+using System.Text;
+using BenchmarkDotNet.Running;
 using IronRure;
+using NetRegex = System.Text.RegularExpressions.Regex;
 
 namespace Alice
 {
     class Program
     {
         static void Main(string[] args)
         {
-            var exePath = Assembly.GetEntryAssembly().Location;;
+            var summary = BenchmarkRunner.Run<AliceBenchTask>();
+        }
+
+        static void Main2(string[] args)
```

Comment:
I'd be happy to just get rid of the old benchmarking code. It will still be in Git if we need it.

Reply:
Sure